packages/web/src/pages/settings-page/components/desktop/ManagerMode/AccountsYouManageHomePage.tsx

```diff
@@ -37,6 +43,28 @@ export const AccountsYouManageHomePage = ({
   const [rejectManagedAccount, rejectResult] = useRemoveManager()
   const { toast } = useContext(ToastContext)
 
+  useEffect(() => {
+    if (managedAccounts == null) return
+    if (
+      pending != null &&
```

Comment:
Might be a good idea to move this into a helper function. I'm actually surprised we don't already have something in utils somewhere to get the current query params...